v0/destinations/tiktok_ads/transform.js

```diff
@@ -37,10 +37,39 @@ function checkIfValidPhoneNumber(str) {
   return regexExp.test(str);
 }
 
+const getContents = (message, payload) => {
+  let contents = payload.properties?.contents || [];
```

Comment:
we should support either `contents` or `products` but not both. Ideally according to Ecomm spec, we should consider `products` but for backward compatibility support `contents` too. Posted query to David, let's do the change post confirmation

Reply:
Done